CONTRIBUTING.md

```diff
@@ -1,601 +1,501 @@
-    - [Configure Eclipse](#configure-eclipse)
-  - [Project setup](#project-setup)
-    - [Source code](#source-code)
-    - [Setup Target Platform](#setup-target-platform)
-    - [Launch Portfolio Performance](#launch-portfolio-performance)
-    - [Build with Maven](#build-with-maven)
-  - [Contribute code](#contribute-code)
+- [Quick Start](#quick-start)
+- [Development Setup](#development-setup)
+  - [Eclipse IDE Setup](#eclipse-ide-setup)
+- [Project Architecture](#project-architecture)
+- [Building the Project](#building-the-project)
+  - [With Eclipse](#with-eclipse)
+  - [With Maven](#with-maven)
+- [Contributing Code](#contributing-code)
+  - [Before You Start](#before-you-start)
+  - [Writing Code](#writing-code)
+  - [Submitting Changes](#submitting-changes)
+- [Specialized Development Guides](#specialized-development-guides)
+  - [PDF Importers](#pdf-importers)
+  - [Interactive Flex Query Importers](#interactive-flex-query-importers)
   - [Translations](#translations)
-    - [Label descriptions](#label-descriptions)
-  - [Images, Logo and color](#images-logo-and-color)
-    - [Format and size](#format-and-size)
-    - [Color code and status](#color-code-and-status)
-  - [Interactive-Flex-Query Importer](#interactive-flex-query-importer)
-    - [Source location](#source-location)
-    - [Imported transactions](#imported-transactions)
-    - [Naming conventions for detected elements](#naming-conventions-for-detected-elements)
-    - [Test cases](#test-cases)
-  - [PDF importer](#pdf-importer)
-    - [Debug information](#debug-information)
-    - [Source location](#source-location-1)
-    - [Imported transactions](#imported-transactions-1)
-    - [Anatomy of a PDF importer](#anatomy-of-a-pdf-importer)
-    - [Naming conventions for detected values](#naming-conventions-for-detected-values)
-    - [Auxiliary classes](#auxiliary-classes)
-    - [Formatting of PDF importer](#formatting-of-pdf-importer)
-    - [Test cases](#test-cases-1)
-    - [Regular expressions](#regular-expressions)
-  - [Trade calendar](#trade-calendar)
-    - [Source location](#source-location-2)
-    - [Anatomy of a trade calendar](#anatomy-of-a-trade-calendar)
-    - [Test cases](#test-cases-2)
+  - [Trade Calendars](#trade-calendars)
+  - [Images and Icons](#images-and-icons)
+- [Getting Help](#getting-help)
+- [Appendix](#appendix)
+  - [Regular Expression Reference](#regular-expression-reference)
+  - [Color Code Reference](#color-code-reference)
 
-## Development setup
+## Quick Start
 
+Portfolio Performance is an Eclipse RCP application for tracking and analyzing investment portfolios. To contribute:
 
-### Install Eclipse
+1. **Fork and clone** the repository
+2. **Set up your development environment** (Eclipse IDE required for RCP development)
+3. **Check the architecture** to understand the codebase structure
+4. **Make your changes** following our coding standards
+5. **Test thoroughly** and submit a pull request
```

Comment:
Not sure if it is helpful to detail this on this level... I want developers that have a basic understanding (or a willing to educated themselves how to contribute on Github).